x/auth/client/cli/sign.go

```diff
@@ -52,6 +53,9 @@ recommended to set such parameters manually.`,
 	cmd.Flags().Bool(flagValidateSigs, false, "Print the addresses that must sign the transaction, "+
 		"those who have already signed it, and make sure that signatures are in the correct order.")
 	cmd.Flags().Bool(flagOffline, false, "Offline mode. Do not query local cache.")
+	cmd.Flags().StringP(flagOutfile, "O", "-",
+		"The document will be written to the given file. If - is used as file, "+
```

Comment:
What about if no `-O` flag specified, write to stdout? That seems more obvious.

Reply:
Yeah, I borrowed it from `wget`. Seems fair though, will change it